src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="Photo of Apple iMac 27"
+      >
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">
+        <span>Product code:</span>
+        <span data-id="id">195434</span>
+      </div>
+
+      <div class="card__rating">
+
+        <div class="stars stars--4">
+          <div class="stars__star star1"></div>
+          <div class="stars__star star2"></div>
+          <div class="stars__star star3"></div>
+          <div class="stars__star star4"></div>
+          <div class="stars__star star5"></div>
+        </div>
+
+        <div class="card__reviews">
+          <span>Reviews:</span>
+          <span>5</span>
+        </div>
+      </div>
+
+      <div class="card-price">
+        <span class="card-price__text">Price:</span>
+        <span class="card-price__amount">$2,199</span>
+      </div>
+
+      <button class="button" data-qa="hover">BUY</button>
```

Comment:
The `button' block is not needed, as it is written in the instructions for the task. We should have only two BEM blocks. Of course it's okay to do like this, but we should follow requirements
![image](https://user-images.githubusercontent.com/73229127/185764292-adcc0fba-3d66-467b-acce-d55df1132138.png)

Reply:
Thanks.
I wrote exactly the same before BEM)

Reply:
Same here, it's an element, not a BEM-block

Reply:
Sorry. It was a misunderstanding appear with the level of my English.
Already done!)